README.md

```diff
@@ -28,9 +28,8 @@ sudo apt install cmake bison
 You can use the standard iceoryx cmake build approach with an activated `-DDDS_STACK=CYCLONE_DDS`
 or `-DDDS_STACK=FAST_DDS` switch.
 ```sh
-cmake -Bbuild -DBUILD_TEST=ON -DDDS_STACK=CYCLONE_DDS # DDDS_STACK=FAST_DDS
-cd build
-make
+cmake -Bbuild -DBUILD_TEST=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DDDS_STACK=CYCLONE_DDS # -DDDS_STACK=FAST_DDS
```

Comment:
Could you please remove the `-DDDS_STACK=FAST_DDS` comment at the end of the line and add some documentation after the code block that there are currently two values supported for the `DDS_STACK` cmake variable, one being `CYCLONE_DDS` and the other one `FAST_DDS`. With `CYCLONE_DDS` being the default.